### DISCUSSION TITLE
Generate CRUD screen from some postgreSQL table not possible

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7589] [Help] [answered]
In my postgreSQL DB, I have several tables.
For some of them, I am able to generate the CRUD screen.
For some others, I am not : I just can't check the table in the table list.

My opinion is that it is impossible for tables with foreign keys, is it ?

**Comment by NicolasFabre81** [comment, marked as answer]
Now it is possible, there is maybe a bug ... I think I just refreshed the page...

**Reply by melohagan** [reply]
If you are able to replicate, please do raise a bug with the reproduction steps!
